components/wysiwyg.vue

```diff
@@ -73,10 +73,10 @@ export default {
       selectedRange: null,
       QuillComponent: null,
       fontsArray: [
-        "1.00rem", "0.25rem", "0.50rem", "0.75rem", "1.25rem", "1.50rem", "1.75rem", "2.00rem",
-        "2.25rem", "2.50rem", "2.75rem", "3.00rem", "3.25rem", "3.50rem", "3.75rem", "4.00rem",
-        "4.25rem", "4.50rem", "4.75rem", "5.00rem", "5.25rem", "5.50rem", "5.75rem", "6.00rem",
-        "6.25rem", "6.50rem", "6.75rem", "7.00rem", "7.25rem", "7.50rem", "7.75rem", "8.00rem"
+        false, "0.25rem", "0.5rem", "0.75rem", "1rem", "1.25rem", "1.5rem", "1.75rem", "2rem",
```

Comment:
what is this false value first ?

We need 1rem by default and it should be correctly ordered in the list

Reply:
The false value is handled by quill as the default value, which does not set any inline styling to the text.
I suggest to keep and not force a 1rem default inline styling with `style="font-size: 1rem;"`, first to keep the possibility of removing font size styles and second to keep supporting normal text with no inline styling

Reply:
Got you but then put in place of the 1rem value as normal is actually equivalent to 1rem, isn't it ?

Reply:
Correct as p tags has 1 rem default values